pkg/controller/lifecycle/cnr.go

```diff
@@ -308,6 +311,12 @@ func (cl *CNRLifecycle) updateOrCreateCNR(node *corev1.Node) error {
 		if err != nil && !errors.IsAlreadyExists(err) {
 			return fmt.Errorf("failed to create cnr %s: %v", cnr.Name, err)
 		}
+		if errors.IsAlreadyExists(err) {
+			cnr, err = cl.client.InternalClient.NodeV1alpha1().CustomNodeResources().Get(cl.ctx, node.Name, metav1.GetOptions{ResourceVersion: "0"})
```

Comment:
Should it also fix `updateOrCreateCNC` of cnc lifecycle like this?

Reply:
Yes, cnc will be fixed in a later pr.